src/BasisTheory.net/Common/Responses/TokenResponse.cs

```diff
@@ -8,39 +8,39 @@ namespace BasisTheory.net.Common.Responses
     public abstract class TokenResponse
     {
         [JsonProperty("id", Order = -3)]
-        [JsonPropertyName("id")]
+        [JsonPropertyName("id"), JsonPropertyOrder(-3)]
```

Comment:
NBD, but negative ordering values looks funny. Maybe we should reindex to start at 0.

Reply:
We start with negative ordering to allow all un-ordered properties (default to `0`) to show in the middle.